std_msgs/Header header
geometry_msgs/Pose pose
geometry_msgs/Point min
geometry_msgs/Point max
geometry_msgs/Vector3 dimensions
visualization_msgs/Marker marker
float64 area
float64 volume
uint32 label
